### ISSUE TITLE
Release including "Callable"

### ISSUE THREAD

**Issue by timkpaine** [ipython/traitlets#554]
Is it possible to do a new release to include the "Callable" Traitlet

**Comment by rmorshea** [comment]
Unfortunately `master` has progressed a great deal beyond where the latest release has been. Given how core `traitlets` is to the Jupyter ecosystem there's been concern about what might happen if we created a release based on `master`. Thus, until testing is done to ensure `master` does not cause significant issues, we've been creating releases off https://github.com/ipython/traitlets/tree/4.3.x.

TLDR; make a PR containing the back-ported feature against the `4.3.x` branch

**Comment by timkpaine** [comment]
@rmorshea https://github.com/ipython/traitlets/pull/555

**Comment by rmorshea** [comment]
This was successfully merged and is awaiting a release from `4.3.x`
